Bottom: 7ef03c13f649bcf64178d7acf16a1b80b80392a3
Top:    bf5f2e7e266b4d0262fc91e39b6aad452425c5ef
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 15:28:54 +0000

Refresh of add-some-annotations-to-help-o

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 4808d9b..d1bc197 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -870,7 +870,7 @@ M_POLYAREA_intersect (jmp_buf * e, POLYAREA * afst, POLYAREA * bfst, int add)
 	if (curcA->Flags.status == ISECTED)
 	  {
 	    the_list = add_descriptors (curcA, 'A', the_list);
-	    if (UNLIKELY (the_list) == NULL)
+	    if (UNLIKELY (the_list == NULL))
 	      error (err_no_memory);
 	  }
     }
